pkg/internal/vault/vault.go

```diff
@@ -355,3 +340,29 @@ func (v *Vault) requestTokenWithKubernetesAuth(client Client, kubernetesAuth *v1
 func (v *Vault) Sys() *vault.Sys {
 	return v.client.Sys()
 }
+
+func extractCertificatesFromVaultCertificateSecret(secret *certutil.Secret) (cert []byte, ca []byte, err error) {
```

Comment:
```suggestion
func extractCertificatesFromVaultCertificateSecret(secret *certutil.Secret) ([]byte, []byte, error) {
```

named return variables should be avoided in Go now